components/hab/install.ps1

```diff
@@ -23,6 +23,32 @@ param (
 
 $ErrorActionPreference="stop"
 
+Function Get-Version-Pcio($version, $channel) {
```

Comment:
I was having trouble shortening `packages.chef.io` into something that was still meaningful.  I went this route because the longer term goal is to migrate old release out of bintray and then this can revert to `Get-Version`.